src/main/external-resources/renderers/Panasonic-VieraAS650.conf

```diff
@@ -17,7 +17,7 @@ RendererIcon = Panasonic-VieraAS650.png
 
 UserAgentAdditionalHeader = X-PANASONIC-Registration
 UserAgentAdditionalHeaderSearch = REFOVFYAAAAAAAAAAAAAAAA
-UpnpDetailsSearch = \d{2}AS650
+UpnpDetailsSearch = \\d{2}AS650
```

Comment:
@Sami32 I wrote a new unit test for this detection which shows it works either way (`\d` or `\\d`) https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1316/commits/85651a905055c6c7426544b607291203208b1962#diff-dc6893ca8f996b0b646af9054ea477f2